doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -2,64 +2,52 @@
  Mount CephFS using Kernel Driver
 =================================
 
-Prerequisite
-------------
-Before mounting CephFS, copy the Ceph configuration file and keyring for the
-CephX user that has CAPS to mount MDS to the client host (where CephFS will be
-mounted and used) from the host where Ceph Monitor resides. Please note that
-it's possible to mount CephFS without conf and keyring, but in that case, you
-would have to pass the MON's socket and CephX user's secret key manually to
-every mount command you run.
+To make a choice between kernel and FUSE mounted CephFS and to go through
+the prerequisites required by both read `Mount CephFS: Prerequisites`_ page.
```

Comment:
I think we can forgo this warning. Just very briefly document what the kernel driver is. Something like:

> The CephFS kernel driver is part of the Linux kernel. It allows mounting CephFS as a regular file system with native kernel performance. It is the client of choice for most use-cases.

Reply:
> I think we can forgo this warning.

I think your point about pointing user to super mount doc is really valid especially since they can directly land to kernel's or fuse's page.